codingTest_김준우/가장 적은 수 제거하기 .js

```diff
@@ -0,0 +1,21 @@
+function solution(arr) {
+  let answer = [];
+  let firstIndex = 0;
```

Comment:
```suggestion
  let minNum= 0;
```
문제에서 볼 때 가장 작은 수를 찾는데 사용되는 변수이기 때문에 ```minNum```이 더 적절해 보이네요.